azurerm/data_source_kubernetes_cluster.go

```diff
@@ -244,6 +244,23 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"windows_profile": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"admin_username": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"admin_password": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
```

Comment:
since this is the data source, as this field isn't returned it's simplest just to remove this field from the schema for now